src/Dynode/controller/network.ts

```diff
@@ -42,5 +43,19 @@ export default class NetworkController {
       stage.debug[`node-${nodeModel.name}`] = nodeUI; // References to NodeUI instances for debugging purposes
     }
     this.model.addEventListener('addNode', addNode);
+    this.view.addEventListener('menu', (e: VEvent<StageView>) => {
+      if (e.detail === undefined) throw new Error('Menu command was not given.');
+      switch (e.detail) {
+        case 'core.network.resolve': {
+          this.model.resolve();
+          break;
+        }
+        default: {
+          throw new Error(`Command ${e.detail} not found.`);
+          break;
+        }
+      }
+      this.model.resolve();
```

Comment:
- If the command is not recognized just ignore it. Or any custom menu command will have to be supported by the network controller.
- That second `this.model.resolve()` is redundant, delete it.